docs/roman/source_detection/arguments.rst

```diff
@@ -0,0 +1,55 @@
+Arguments
+=========
+The source detection fitting step has several arguments. These can be specified
+by the user by passing them to the step in a Python session, or setting them
+in a parameter file.
+
+* ``--coverage_mask``: True/False to mask pixels with DO_NOT_USE flags in their
+                       dq array. Default is True.
```

Comment:
Just out of curiosity, can you envision cases where we don't want this set to True?

Reply:
I can't think of any. I think my original idea was to allow someone to pass in an array of their own mask, but I'll just remove it as an option for now.